clients/tv/src/index.tsx

```diff
@@ -7,7 +7,12 @@ import ConsoleExecutor from './executors/ConsoleExecutor';
 
 // since speechSynthesis.speak() without user activation is no longer allowed
 // executeFmCommand('speech', 'Cześć!') can be triggered from console for now
-window['executeFmCommand'] = ConsoleExecutor.executeFmCommand
+declare global {
+    interface Window {
+        executeFmCommand: Object;
```

Comment:
Tylko, że to nie prawda - `executeFmCommand` jest typu `(string, string[]) => void`, a nie `Object`.
Chociaż z tego co mówiłeś to jest tymczasowe, do testów, więc tutaj nawet na `// @ts-ignore` bym się zgodził.

Reply:
zrobiłem ignore'a w takim wypadku

Reply:
W porządeczku :)